app/messages/farmer-apply.js

```diff
@@ -0,0 +1,18 @@
+import { v4 as uuidv4 } from 'uuid'
+import { NotifyClient } from 'notifications-node-client'
+
+const farmerApply = async (message) => {
+  const notifyClient = new NotifyClient(process.env.NOTIFY_API_KEY)
+
+  await notifyClient.sendEmail(
```

Comment:
I think as the comms message isn't specific it would be better to call Notify using fields from the message.

This will apply to:
* Template ID
* Email Address
* Personalisation